staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -226,12 +226,21 @@ else {
 }
 
 if ($enableAll -or $enableSnapshotSize) {
-  Write-Host "Collecting container snapshot size using DU tool"
-  if (Test-Path "C:\aks-tools\DU\du.exe") {
-    C:\aks-tools\DU\du.exe /accepteula
-    C:\aks-tools\DU\du.exe -l 1 C:\ProgramData\containerd\root\io.containerd.snapshotter.v1.windows\snapshots\ > "$ENV:TEMP\$timeStamp-du-snapshot-folder-size.txt"
-    $paths += "$ENV:TEMP\$timeStamp-du-snapshot-folder-size.txt"
+  Write-Host "Collecting actual size of snapshot (without sparse file sizes included)"
+
+  $snapshotPath = "C:\ProgramData\containerd\root\io.containerd.snapshotter.v1.windows\snapshots\"
+  $snapshotSizesResultFilePath = "$ENV:TEMP\$timeStamp-all-snapshot-folder-size.txt"
+  $listOfSnapshotFolders = Get-ChildItem $snapshotPath | Where-Object {$_.PSIsContainer -eq $true} | Sort-Object
+  foreach ($i in $listOfSnapshotFolders) {
+  	$totalSize = 0   
+  	Get-ChildItem -Path $i.FullName -recurse -Attributes !SparseFile | Where-Object {$_.PSIsContainer -eq $false} | ForEach-Object {
+  		$totalSize = $totalSize + $_.Length
+  	}
+  	$output = "Sum of " + $i.FullName + " is " + ($totalSize/1MB) + "MB"
+    Add-Content -Path $snapshotSizesResultFilePath -Value $output
```

Comment:
This should print the total size of the entire directory in addition to the size of each snapshot.